pkg/exprhelpers/exprlib.go

```diff
@@ -116,6 +133,56 @@ func Init(databaseClient *database.Client) error {
 	return nil
 }
 
+func RegexpCacheInit(filename string, CacheCfg types.DataSource) error {
+	if dataFileRegexCache == nil {
+		dataFileRegexCache = make(map[string]gcache.Cache)
```

Comment:
I would allocate empty maps at the package level (less likely to have nil dereference by mistake) which costs 48bytes